### ISSUE TITLE
Interactive demos gallery?

### ISSUE THREAD

**Issue by vincerubinetti** [3b1b/3Blue1Brown.com#56]
Perhaps have a gallery of interactive/react apps that looks sorta like this:

![image](https://user-images.githubusercontent.com/8326331/120381467-740ed200-c2f0-11eb-8b92-6bbb2fdedb4c.png)

Perhaps it would live as another tab next to "featured" "by topic" and "by date".

Something to workshop once we have more interactive apps.

**Comment by 3b1b** [comment]
>  Something to workshop once we have more interactive apps.

Agreed.  In fact, I'll tag this as low priority for just that reason.

**Comment by vincerubinetti** [comment]
This can be achieved with an `interactive` tag on the lessons that have one, and we can mark them with a special icon or something in the existing search results, in the same way we do for having a readable version.

I'll set this up as part of my fixing the legacy bugs with the interactives, which I've already tagged myself on for the relevant issues.
